knimin/handlers/ag_edit_barcode.py

```diff
@@ -40,7 +43,8 @@ def post(self):
                         sites_sampled=None, details=None,
                         environments_sampled=None,
                         logins=None, currentuser=self.current_user)
-        except:
+        except Exception as e:
+            print str(e)
```

Comment:
I had it in to get the error into the logs, since right now it's ignored, but I can remove it if wanted.